doc/source/user_guide/ug_implementation.rst

```diff
@@ -199,6 +199,9 @@ i.e. f\_fbio, except they are averaged by ice area.
    "f\_faero\_atm", "atmospheric aerosol deposition flux", "faero\_atm", "kg m\ :math:`^{-2}` s\ :math:`^{-1}`"
    "f\_faero\_ocn", "aerosol flux from ice to ocean", "faero\_ocn", "kg m\ :math:`^{-2}` s\ :math:`^{-1}`"
    "f\_aero", "aerosol mass (snow and ice ssl and int)", "aerosnossl, aerosnoint,aeroicessl, aeroiceint", "kg/kg"
+   "f\_fiso\_atm", "atmospheric isosol deposition flux", "fiso\_atm", "kg m\ :math:`^{-2}` s\ :math:`^{-1}`"
+   "f\_fiso\_ocn", "isosol flux from ice to ocean", "fiso\_ocn", "kg m\ :math:`^{-2}` s\ :math:`^{-1}`"
+   "f\_iso", "isotope mass (snow and ice)", "aerosno, aeroice", "kg/kg"
```

Comment:
the notation f_ usually indicates a flux, not mass.  I see this is a problem for aero too.  Is there a reason that f_ is used for aero and isotopes?  The entries in this file need to be cleaned up, e.g. 'isosol' and change aerosno, aeroice to isosno, isoice.